lisa/tools/windows_feature.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.operating_system import Windows
+from lisa.tools.powershell import PowerShell
+from lisa.util import LisaException
+
+
+# WindowsFeature management tool for Windows Servers.
+# It can install, uninstall, and check the status of Windows features.
+# Hyper-V, DHCP etc. are examples of Windows features.
+# This tool uses PowerShell to manage Windows features.
+# Not supported on PC versions like Windows 10, 11 etc.
+class WindowsFeature(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        assert isinstance(self.node.os, Windows)
```

Comment:
Add error message, not just assert, but no error message.

Reply:
Details added.